classes/redis_store.php

```diff
@@ -113,14 +114,28 @@ protected function bootstrap_redis() {
 
         if (!class_exists('Redis')) {
             throw new \coding_exception('Redis class not found, Redis PHP Extension is probably not installed');
-        }
-        if (empty($CFG->auth_saml2_redis_server)) {
-            throw new \coding_exception('Redis connection string is not configured in $CFG->auth_saml2_redis_server');
+        }        
+        if (!empty($CFG->auth_saml2_redissentinel_servers) && !empty($CFG->auth_saml2_redissentinel_group)) {
```

Comment:
These really need to be plugin scope config items not globals please, its a very simple change. Get the config using get_config and set it in config.php like:

```
$CFG->forced_plugin_settings['auth_saml2']['foobar'] = 0;
```

This also needs to be documented somewhere like the readme file